translations/strings.yaml

```diff
@@ -337,23 +337,55 @@ telemetryPolicyView:
   description: We strive to provide you with choices and collect only the data we need to improve Mozilla VPN. Sharing this data with Mozilla is optional.
   question: Allow Mozilla VPN to send technical, interaction, and campaign and referral data to Mozilla?
 
+genericError:
+  unexpected: Unexpected error
 
+# Connection Info Screen
 connectionInfo:
   labelBitps:
-    value: bit/s
+    value: bits
     comment: Bits per second
   labelKbitps:
-    value: kbit/s
+    value: kbits
     comment: Kilobits per second
   labelMbitps:
-    value: Mbit/s
+    value: Mbits
     comment: Megabits per second
   labelGbitps:
-    value: Gbit/s
+    value: Gbits
     comment: Gigabits per second
   labelTbitps:
-    value: Tbit/s
+    value: Tbits
     comment: Terabits per second
+  labelPing:
+    value: Ping
+    comment: Latency of the connection
+  unitPing:
+    value: ms
+    commnent: milliseconds
+  loadingIndicatorLabel:
+    value: Testing speed
```

Comment:
Why is there no ellipsis?

Reply:
I thought it might make sense to add the ellipsis where the string is used so it’s not replaced by three dots by mistake? Happy to add it to the string.

Reply:
Oh — I guess there is the possibility that different locales use a different typographic sign? I’ll just add it here.

Reply:
[Never hard-code](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#dont-hardcode-characters) punctuation signs. That's up to locales to choose what to use and how.

Reply:
Example: [Japanese](https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/blob/main/ja/mozillavpn.xliff#L140).